src/uu/ls/src/ls.rs

```diff
@@ -366,24 +367,36 @@ impl Config {
             })
             .or_else(|| termsize::get().map(|s| s.cols));
 
+        let show_control = if options.is_present(options::HIDE_CONTROL_CHARS) {
+            false
```

Comment:
it is possible to add a test for this case too?

Reply:
Ah of course, forgot about that because it is the default, but I added some more tests now

Reply:
as always, we forgot to test the most obvious stuff :)